static-mustache/src/main/java/com/github/sviperll/staticmustache/GenerateRenderableAdapterProcessor.java

```diff
@@ -209,6 +210,14 @@ private void writeRenderableAdapterClass(TypeElement element, AnnotationMirror d
             SwitchablePrintWriter switchablePrintWriter = SwitchablePrintWriter.createInstance(stringWriter);
             try {
                 FileObject templateBinaryResource = processingEnv.getFiler().getResource(StandardLocation.CLASS_OUTPUT, "", templatePath);
+                //On some environments (e.g. NetBeans with compile on save) this won't resolve the correct path
+                //thus we try another route
+                if (!new File(templateBinaryResource.toUri()).exists()) {
+                    File abs = new File(GenerateRenderableAdapterProcessor.class.getClassLoader().getResource(templatePath).getFile());
+                    if (abs.exists()) {
+                        templateBinaryResource = new FileAdapter(abs);
```

Comment:
As per suggestions from above. I think it's better to use `Path`-API than `File`-API and it's better to path charset to `FileAdapter`-constructor.

```suggestion
                        templateBinaryResource = new FileAdapter(abs, templateCharset);
```